times_reader/transforms.py

```diff
@@ -23,21 +23,212 @@
     "CSet_CD",
 }
 
+# Specify a list of aliases per TIMES attribute
+aliases_by_attr = {
```

Comment:
Will this, and the other dictionaries below, be eventually replaced by the JSON file you're working on in #39 ?

Reply:
By a different json, I think. The one in #39 focuses on the valid fields for the input tables. We should probably have a separate one focusing on attributes and their aliases. What do you think?

Reply:
Sure that sounds good. We can have a folder of all this kind of meta information and add doc strings to the top of each file explaining what is defined there.